### ISSUE TITLE
Timestamp in History

### ISSUE THREAD

**Issue by joshuaellinger** [COVID19Tracking/website#472]
[I am posting this to Website to encourage you to decide how to display a timestamp on each history row as part of the table redesign -- I did not feel comfortable adding it as a comment under redesign because it felt a little off-topic - Josh]

### Example of the Problem

We have a confusing issue where data comes in after the 2nd shift where we push to history and it gets updated in a way that makes it look like we are being inconsistent.  The actual data is right (at least everywhere I've looked) but it looks wrong if you aren't deeply knowledgeable about our process.

Here is an example from CA:

![image](https://user-images.githubusercontent.com/2719463/78960637-43e4ee00-7ab4-11ea-9ff3-c780dc639655.png)

The data came in late so the 4/7 shift initially published the result with the previous day's value.

The values weren't updated until the second shift on 4/8.  They entered a update time of 4/7 18:00 to update the history.  If you display "(at 6:00PM)" in the history row, the data makes sense and it is obvious to the viewer that the state reported late rather instead of us making a mistake.

![image](https://user-images.githubusercontent.com/2719463/78960902-1b112880-7ab5-11ea-9a3c-4d0b9e75e30f.png)

### API Dependencies

The timestamp is recorded in the Checks tab but it is not accessible to the web site (nor should it be used directly).  We need to enhance the API to give you this date per history row.

Once that is available, I don't have a strong opinion how you display it.  You could only display the cases where it changed after the second shift in a mouse over and it would be clear.  You could also just put a * to indicate a change.  But I think the clearest thing to do is always display it.

**Comment by schwartzadev** [comment]
Shout-out to @joshuaellinger for the _super_ detailed PR -- thank you!

@webmasterkai  -- would you be able to expose that last updated field on the API?

I think we should try to resolve this ASAP

**Comment by muamichali** [comment]
Not stale, it’s in progress https://covid-tracking.slack.com/archives/C011AP48CS3/p1588174109166500

**Comment by kevee** [comment]
Now that we have `lastUpdateET` in the daily feed, I can add that instead of `dateChecked`. Do you want to just add time when it's not midnight?

![image](https://user-images.githubusercontent.com/270536/81208795-98896500-8f9d-11ea-8a67-a071bedced25.png)

**Comment by joshuaellinger** [comment]
Displaying the time part of lastUpdateET would be misleading if the edit occurred the following morning.  For example,  lastUpdateET for the 3/25 row in your example could have been updated at 3/26 at 1PM.  In your second view, you would show 3/25 1PM (I think).

A view would wonder why the values are bigger than all of the screen shots and we're no better than when we started.

So my suggestion is to optimize for the no-change case.  I the lastUpdateDT > the 2nd push time for the day (example 3/25 at 5PM ET), then just show red * .  When you mouse over it show a tool-tip saying "Updated on 3/36 at 1PM ET".

**Comment by joshuaellinger** [comment]
![image](https://user-images.githubusercontent.com/2719463/81214142-02edd580-8f9d-11ea-8e75-b7aadd09b73e.png)

**Comment by kevee** [comment]
OK the UI is up to the design team, but I'll see what we can come up with.

**Comment by joshuaellinger** [comment]
Of course.  Just wanted to make sure you were aware of the next day issue.  Hate to clutter up the site for something that is mainly an intermittent issue for the West Coast.
